src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -1,145 +1,99 @@
 package com.wikia.webdriver.common.core.url;
 
-import com.wikia.webdriver.common.core.configuration.Configuration;
+import static com.wikia.webdriver.common.core.configuration.Configuration.DEFAULT_LANGUAGE;
+import static com.wikia.webdriver.common.core.configuration.Configuration.getEnv;
+import static com.wikia.webdriver.common.core.configuration.Configuration.getEnvType;
+import static com.wikia.webdriver.common.core.configuration.Configuration.getForceHttps;
+import static com.wikia.webdriver.common.core.configuration.Configuration.getForceLanguageInPath;
+import static com.wikia.webdriver.common.core.configuration.Configuration.getWikiLanguage;
+import static com.wikia.webdriver.common.core.configuration.Configuration.getWikiName;
+
 import com.wikia.webdriver.common.core.configuration.EnvType;
 import okhttp3.HttpUrl;
-import org.apache.commons.lang.StringUtils;
 import org.openqa.selenium.WebDriverException;
 
-public class UrlBuilder {
+public class UrlBuilder extends BaseUrlBuilder {
 
 
-  protected String env;
-  protected EnvType envType;
+  private final String wikiName;
   private Boolean forceHttps;
   private Boolean forceLanguageInPath;
-  protected static final String HTTP_PREFIX = "http://";
+  private String language;
 
-  public UrlBuilder() {
-    this.env = Configuration.getEnv();
-    this.envType = Configuration.getEnvType(this.env);
-    this.forceHttps = Configuration.getForceHttps();
-    this.forceLanguageInPath = Configuration.getForceLanguageInPath();
+  private UrlBuilder(String wiki, String env, Boolean forceHttps, Boolean forceLanguageInPath, String language) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "UrlBuilder" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)